recipes-examples/celebrity-face-match/demo-celebrity-face-match_0.4.bb

```diff
@@ -16,28 +16,20 @@ BRANCH = "main"
 SRCREV = "89b6ca51d233fb1eabe0b80be192081dea017245"
 
 S = "${WORKDIR}/git"
-INSTALL_DIR = "${D}${datadir}/${PN}"
 
-inherit systemd
+DEPENDS += "gtk+3"
 
-SYSTEMD_SERVICE:${PN} = "demo-celebrity-face-match.service"
-
-do_configure[noexec] = "1"
-do_compile[noexec] = "1"
+inherit meson pkgconfig systemd
 
-do_install() {
-    install -Dm 0644 ${WORKDIR}/demo-celebrity-face-match.service ${D}${systemd_system_unitdir}/demo-celebrity-face-match.service
+SYSTEMD_SERVICE:${PN} = "demo-celebrity-face-match.service"
 
-    install -d ${D}${datadir}/${PN}
-    for f in ${S}/*.py; do \
-        install -Dm 0644 $f ${D}${datadir}/${PN}/
-    done
+do_install:append() {
+    install -Dm 0644 ${WORKDIR}/${BPN}.service ${D}${systemd_system_unitdir}/${BPN}.service
 }
 
 RDEPENDS:${PN} += " \
     demo-celebrity-face-match-data \
```

Comment:
This and gstreamer need to stay here, for the rest cmp phytec/demo-celebrity-face-match#7 ?
Actually, what about `v4l-utils` as we call `v4l2-ctl`?

Reply:
These are all runtime dependencies and should be left as is.